src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/ComObjectTypeTest.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
This test maps to https://github.com/dotnet/runtime/pull/37402/files#diff-e2f257d8e7f2aa2a43d0b6100a0109a6R2322.

Reply:
I think it would be good to add comments to each test that documents what they are testing.